FC-0068 docs: How-to cleaning by edunext #9 (#714)
openedx/docs.openedx.org: source/educators/how-tos/advanced_features/chage_learner_cohort_assignment.rst

.. _Changing Student Cohort Assignments:

Change Learner Cohort Assignments
##################################

.. tags:: educator, how-to

After your course starts and learners begin to contribute to the course
discussion, each post that they add is visible either to everyone or to the
members of a single cohort. When you change the cohort that a learner is

> assigned to, there are three results.

Reviewer: ```suggestion
assigned to, the following occur:
```

* The learner continues to see the posts that are visible to everyone.

> * The learner sees the posts that are visible to his new cohort.

Reviewer: ```suggestion
* The learner now sees the posts that are visible to their new cohort.
```

> * The learner no longer sees the posts that are visible only to his original

Reviewer: ```suggestion
* The learner no longer sees the posts that are visible only to their original
```
